### PR TITLE
Update pd-configuration-file.md

### PR DESCRIPTION
**Pull request by nolouch**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update pd-configuration-file.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-configuration-file.md [hunk 1]
```diff
@@ -243,6 +243,14 @@ Configuration items related to scheduling
 + The number of Region scheduling tasks performed at the same time
 + Default value: `2048`
 
+### `hot-region-schedule-limit`
```
Comment (TomShawn) [inline pd-configuration-file.md:246]:
```suggestion
### `hot-region-schedule-limit`

```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -243,6 +243,14 @@ Configuration items related to scheduling
...
++ The number of Hot Region scheduling tasks performed at the same time. It is independent with Region scheduling.
```
Comment (TomShawn) [inline pd-configuration-file.md:247]:
```suggestion
+ Controls the hot Region scheduling tasks that are running at the same time. It is independent of the Region scheduling.
```

File: pd-configuration-file.md [hunk 3]
```diff
@@ -243,6 +243,14 @@ Configuration items related to scheduling
...
++ Default value: `4`
+
+### `hot-region-cache-hits-threshold`
++ The threshold used to set the number of minutes required to identify the hotspot region. Only after the region is in the hotspot state for more than this number of minutes can it participate in hot region scheduling
```
Comment (TomShawn) [inline pd-configuration-file.md:251]:
```suggestion

+ The threshold used to set the number of minutes required to identify a hot Region. PD can participate in the hotspot scheduling only after the Region is in the hotspot state for more than this number of minutes.
```

**Review by TomShawn — COMMENTED** [review]
To make the description consistent with https://github.com/pingcap/docs/blob/master/pd-control.md#config-show--set-option-value--placement-rules.

**Review by Ishiihara — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**